nirc_ehr/resources/views/populateData.html

```diff
@@ -0,0 +1,445 @@
+                        this.pendingDeletes = 0;
+                        document.getElementById('msgbox').innerHTML = '';
+                        Ext4.each(this.tables, function(table){
+                            if (!table.doSkip) {
+                                document.getElementById('msgbox').innerHTML += '<div>Deleting ' + table.label + '...</div>';
+                                this.deleteHandler(table);
+                            } else {
+                                document.getElementById('msgbox').innerHTML += '<div>Skipping ' + table.label + '</div>';
+                            }
+                        }, this);
+                    }
+                });
+
+                return items;
+            },
+
+            tables: tables,
+            deleteHandler: function(table){
+                if (table.deleteFn){
+                    this[table.deleteFn].call(this);
+                }
+                else {
+                    this.truncate(table.schemaName, table.queryName);
+                }
+            },
+
+            truncate: function (schemaName, queryName) {
+                this.pendingDeletes++;
+                LABKEY.Ajax.request({
+                    url: LABKEY.ActionURL.buildURL("query", "truncateTable.api"),
+                    success: LABKEY.Utils.getCallbackWrapper(this.onDeleteSuccess, this),
+                    failure: LDK.Utils.getErrorCallback({
+                        callback: function (resp) {
+                            document.getElementById('msgbox').innerHTML += '<div class="labkey-error">Error loading data: ' + resp.errorMsg + '</div>';
+                        },
+                        scope: this
+                    }),
+                    jsonData: {
+                        schemaName: schemaName,
+                        queryName: queryName
+                    },
+                    headers: {
+                        'Content-Type': 'application/json'
+                    }
+                });
+            },
+
+            onDeleteSuccess: function(data){
+                var count = data ? (data.affectedRows || data.deletedRows) : '?';
+                this.pendingDeletes--;
+                if (this.pendingDeletes === 0){
+                    document.getElementById('msgbox').innerHTML += '<div>Delete Complete</div>';
+                }
+            },
+
+            populateFromFile: function (schemaName, queryName) {
+                this.pendingInserts++;
+                //records for task forms:
+                var config = {
+                    schemaName: schemaName,
+                    queryName: queryName,
+                    moduleResource: '/data/' + queryName + '.tsv',
+                    success: this.onSuccess,
+                    failure: this.onError,
+                    scope: this
+                };
+
+                importFile(config);
+            },
+
+            populateFormTemplateRecords: function() {
```

Comment:
Should be able to delete this section, since no direct data entry means no use for the templates

Reply:
yes, deleted